File: Python/jumbojavelin.py
'''
 *
 *	Problem: "Jumbo Javelin"
 *
'''

numJavelins = int(input())

total = 0
for length in range(numJavelins):
	total += int(input())
	
fusionLoss = numJavelins - 1
total -= fusionLoss

print(total)

File: Java/jumbojavelin.java
/*
 *	Problem: "Jumbo Javelin"
 */

import java.util.Scanner;

public class jumbojavelin {
	
	/*-----main-----*/
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int numJavelins = sc.nextInt();
		int fusionLoss = numJavelins - 1;
		
		int total = 0;
		while (numJavelins-- > 0) {
			total += sc.nextInt();
		}
		
		total -= fusionLoss;
		System.out.println(total);
	} // end main
	
}